data/data/bootstrap/baremetal/files/usr/local/bin/dhcp-filter.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# It is possible machine-api-operator comes up while the bootstrap is
+# online, meaning there could be two DHCP servers on the network. To
+# avoid bootstrap responding to a worker, which would cause a failed
+# deployment, we filter out requests from anyone else than the control
+# plane.  We are using iptables instead of dnsmasq's dhcp-host because
```

Comment:
One day, we need to get rid of MAC address requirement for Redfish (but we're not there yet)